tests/test_tem_distribution_utils.py

```diff
@@ -0,0 +1,26 @@
+"""Tests for tem_distribution_utils."""
+import pytest
+import torch
+
+from simSPI import tem_distribution_utils
+
+
+def test_constructor():
+    """Test if distribution object is made."""
+    generator = tem_distribution_utils.DistributionGenerator("uniform", (0, 10))
+    assert generator.distribution is not None
+
+    generator = tem_distribution_utils.DistributionGenerator("gaussian", (0, 1))
+    assert generator.distribution is not None
+
+    with pytest.raises(NotImplementedError):
+        tem_distribution_utils.DistributionGenerator("invalid_value", (0, 1))
+
+
+def test_draw_samples_1d():
```

Comment:
I would make these tests more specific. Test max or min of uniform. Test some expectations from many samples from Gaussian (mean near zero, variance near scale**2). There is "sample error" but you can use some tolerance.